styles/Canonical/001-English-words-spelling-suggestions.yml

```diff
@@ -0,0 +1,109 @@
+extends: substitution
+message: "Use British English spelling, '%s' instead of '%s'."
+link: https://docs.ubuntu.com/styleguide/en/#spelling
+ignorecase: true
+level: suggestion
+action:
+  name: replace
+swap:
+  'acknowledgment': acknowledgement
+  'airplane': aeroplane
+  'aluminum': aluminium
+  'analog': analogue
+  'analyze': analyse
+  'annex': annexe
+  'apologize': apologise
+  'armor': armour
+  'artifact': artefact
+  'authorization': authorisation
+  'authorize': authorise
+  'authorizer': authoriser
+  'behavior': behaviour
+  'canceled': cancelled
+  'capitalize': 'capitalize'
+  'catalog': catalogue
+  'center': centre
+  'checker': chequer
+  'cipher': cypher
+  'color': colour
+  'counseling': counselling
+  'counselor': counsellor
+  'defense': defence
+  'dialed': dialled
+  'dialer': dialler
+  'dialog': dialogue
+  'emphasize': emphasise
+  'encyclopedia': encyclopaedia
+  'endeavor': endeavour
+  'enrollment': enrolment
+  'equaling': equalling
+  'favor': favour
+  'favorite': favourite
+  'fiber': fibre
+  'flakey': flaky
+  'flavor': flavour
+  'fulfill': fulfil
+  'furor': furore
+  'generalize': generalise
+  'glycerin': glycerine
+  'gray': grey
+  'harbor': harbour
+  'harmonization': harmonisation
+  'harmonize': harmonise
+  'honor': honour
+  'humor': humour
+  'in side': inside
+  'industrialize': industrialise
+  'inquire': enquire
+  'installment': instalment
+  'italicize': italicise
+  'judgment': judgement
+  'kilometer': kilometre
+  'labeled': labelled
+  'labor': labour
+  'lead': led
+  'learned': learnt
+  'license': licence
+  'lite': light
+  'liter': litre
+  'lodgement': lodgment
+  'maneuver': manoeuvre
+  'meter': metre
+  'modeling': modelling
+  'mold': mould
+  'monolog': monologue
+  'offense': offence
+  'organization': organisation
+  'oriented': orientated
+  'parlor': parlour
+  'percent': per cent
+  'plow': plough
+  'practice': practise
+  'pretense': pretence
+  'program': programme
```

Comment:
This particular word is in our style guide:
https://docs.ubuntu.com/styleguide/en/#spelling

I think it should be OK to leave it because the severity level for the rule is "Suggestion" only.

Reply:
i suspect it is going to be easier to leave this out for now

Reply:
@SecondSkoll , I get what you mean about Vale not understanding "context" per se; but there are sequential type of rules that possibly would address these verb vs noun terms.
But that would be a different ticket altogether.

Thanks!

Reply:
But yes I'll remove this one for now.